### PR TITLE
analytics retention dag

### PR DESCRIPTION
**Pull request by akspha**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: dags/daily_analytics_data_retention.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+params = {
+    'bucket': f"agentiq-{env['ENVIRONMENT']}-assets",
+    'duration_months':  env['DATA_RETENTION_DURATION_MONTHS'],
+}
+
+
+dag = DAG('daily_analytics_data_retention',
+          catchup=False,
+          default_args=default_args,
+          # run every day at 6:30am PST after conversation closure
```
Comment (ryaniq) [inline dags/daily_analytics_data_retention.py:41]: 6:30am PST (9:30am EST)?  When do we close conversations?

**Review by ryaniq — APPROVED** [review]
LG

File: dags/daily_analytics_data_retention.py (hunk 1 above)
Comment (akspha) [reply dags/daily_analytics_data_retention.py:41]: we close at 2 am PST
coredb DRS running at 3:30
dynamo DRS running at 4:30
@ryaniq  should we run all at different times, preferably earlier near 00 hours ? I wanted to keep them separate times from existing batch jobs

Comment (akspha) [reply dags/daily_analytics_data_retention.py]: @ryaniq  changed

**Pull request merged**